### ISSUE TITLE
Camunda 7.9.0 / Spring Boot 2.0.2.RELEASE compatibility

### ISSUE THREAD

**Issue by seletz** [camunda/camunda-bpm-camel#43]
Hi guys,

I'm trying to upgrade a camunda 7.8.0 project to the 7.9.0 release.  As of the release notes, I also upgraded from 1.5.10.RELEASE ro the latest 2.c spring boot release.

The 7.9.0 project I have uses this camel-camunda integration -- it's working great!

However, when I switch to 7.9.0/spring boot 2.0.2.RELEASE, I get the following error:

```
2018-05-29 17:59:26.685 ERROR 11000 --- [  restartedMain] o.s.boot.SpringApplication               : Application run failed

org.springframework.beans.factory.BeanDefinitionStoreException: Failed to process import candidates for configuration class [com.nexiles.camundacamel.CamundaCamelApplication]; nested exception is java.lang.IllegalStateException: Could not evaluate condition on org.apache.camel.spring.boot.health.HealthCheckRoutesAutoConfiguration due to org/springframework/boot/bind/RelaxedPropertyResolver not found. Make sure your own configuration does not rely on that class. This can also happen if you are @ComponentScanning a springframework package (e.g. if you put a @ComponentScan in the default package by mistake)
```

Is 7.9.0/spring boot 2.0 even supported?  Am I missing something before going deeper?

**Comment by seletz** [comment]
Duh, forgot to mention my dependency versions:

        <camunda.version>7.9.0</camunda.version>
        <camunda-bpm-spring-boot-starter.version>3.0.0</camunda-bpm-spring-boot-starter.version>
        <camunda-bpm-camel.version>0.6-SNAPSHOT</camunda-bpm-camel.version>
        <camel.version>2.21.1</camel.version>

I've also created a completely new project from scratch to test if camunda 7.9.0 and spring 2.0.2.RELEASE work and it does.

**Comment by seletz** [comment]
The code of the test project is [here](https://github.com/seletz/camunda-spring-camel)

**Comment by seletz** [comment]
OK it seems that apache camel itself is [not yet spring boot 2.x compatible](https://issues.apache.org/jira/browse/CAMEL-12423).  Well -- I guess I'll wait a bit then.  I'll however update the demo project if apache camel finally is there.

**Comment by hawky-4s-** [comment]
Thanks for the info! I would have guessed the same when looking at the
exception message as it explicitly states camel healthcheck.

**Comment by davidkarlsen** [comment]
It's updated now - can you release an updated camunda-bpm-camel version too please?
https://mvnrepository.com/artifact/org.apache.camel/camel-spring-boot-starter/2.22.2

**Comment by berndruecker** [comment]
Hi @davidkarlsen - could you provide a pull request for it? I can make sure it gets merged and released (if it is sound ;-)) - but it might take a bit until I can look at it myself.

**Comment by berndruecker** [comment]
Still have issues with CI. Checking that tomorrow

**Comment by davidkarlsen** [comment]
What parameters and jdk and maven version did you use?

I am on latest 1.8 maven 3.5.0 and doing "clean install"

**Comment by berndruecker** [comment]
Build also works after switching to JDK 1.8 - but release had a wired error. I have to look into that - but today I am traveling - so I can't investigate (bad WIFI at the moment)

**Comment by hawky-4s-** [comment]
Problem: Maven complains about misaligned javadocs for JDK8 as the parser is now stricter. You should be able to reproduce it the error by running ```mvn javadoc:jar``` locally

**Comment by berndruecker** [comment]
@davidkarlsen Do you have time to fix this?

**Comment by davidkarlsen** [comment]
@berndruecker See #45

**Comment by davidkarlsen** [comment]
Will you do a release?

**Comment by davidkarlsen** [comment]
ah - seems you already did! 👍

**Comment by davidkarlsen** [comment]
Still not on central? http://repo1.maven.org/maven2/org/camunda/bpm/extension/camel/camunda-bpm-camel-spring/

**Comment by berndruecker** [comment]
Still not building due to another problem I have to look into (probably some Maven memory settings).

Von: David J. M. Karlsen <notifications@github.com>
Gesendet: Dienstag, 4. Dezember 2018 11:02
An: camunda/camunda-bpm-camel <camunda-bpm-camel@noreply.github.com>
Cc: Bernd Rücker <bernd.ruecker@camunda.com>; State change <state_change@noreply.github.com>
Betreff: Re: [camunda/camunda-bpm-camel] Camunda 7.9.0 / Spring Boot 2.0.2.RELEASE compatibility (#43)


Still not on central? http://repo1.maven.org/maven2/org/camunda/bpm/extension/camel/camunda-bpm-camel-spring/

**Comment by davidkarlsen** [comment]
Any success with the memory increase?

**Comment by berndruecker** [comment]
No  so far it is a bit wired. @hawky-4s- volunteered to look at it as soon as he has some spare time. But it looks that there is a 0.6 release available via our Nexus - unfortunately not on Maven Central. Sorry for the delay.

**Comment by davidkarlsen** [comment]
Is that an error? Do you plan to sync with central?

**Comment by berndruecker** [comment]
It is an error - yes

**Comment by berndruecker** [comment]
@hawky-4s- did fix this! Now we have a proper 0.6 release out there.
